docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx

```diff
@@ -73,28 +73,18 @@ There are two methods for installing Stencil CLI and its dependencies on Windows
 
 
 ### Method 1: Install dependencies using Chocolatey
-If you're not comfortable manually installing and configuring Python and Node.js on Windows, or if you prefer an easy installation option, use the [Chocolatey package manager](https://chocolatey.org/) to install Stencil CLI's dependencies. To do so, [open PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/getting-started/starting-windows-powershell?view=powershell-6) as an administrator, and run the following commands:
+If you're not comfortable manually installing and configuring Python and Node.js on Windows, or if you prefer an easy installation option, use the [Chocolatey package manager](https://chocolatey.org/install) to install Stencil CLI's dependencies. To do so, [open PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/getting-started/starting-windows-powershell?view=powershell-6) as an administrator, and run the following commands:

```

Comment:
```suggestion
If you're not comfortable manually installing and configuring Python and Node.js on Windows, or if you prefer an easy installation option, use the [Chocolatey package manager](https://chocolatey.org/install) to install Stencil CLI's dependencies.  Python and C++ build tools are required to compile some dependencies, but you may already have them installed. The troubleshooting section contains information about which versions Stencil currently uses. ... (run into errors go there)

To install Stencil CLI's dependencies, [open PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/getting-started/starting-windows-powershell?view=powershell-6) as an administrator, and run the following commands:

```

Reply:
I will trust the engineer's testing of the installation instructions and not add this clarification.